### PR TITLE
feat(ng-ovh-payment-method): update adyen env config for india

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #DTRSD-122255
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

## Description

Update Adyen environment config param for India subs.

### CODE REVIEW

File: packages/components/ng-ovh-payment-method/src/components/integration/component/adyen/controller.js [hunk 1]
```diff
@@ -253,4 +256,10 @@ export default class OvhPaymentMethodIntegrationComponentAdyenCtrl {
       callbackParamObject,
     );
   }
+
+  getEnvironment() {
+    return this.user.ovhSubsidiary === IN_SUBSIDIARY
```
Comment (darsene) [inline packages/components/ng-ovh-payment-method/src/components/integration/component/adyen/controller.js:261]: Will another subs will added later ? Maybe you should consider using feature flipping instead ?
What do you think ?

Comment (sachinrameshn) [reply packages/components/ng-ovh-payment-method/src/components/integration/component/adyen/controller.js:261]: May be not as the environment is only for IN, but I will consider your suggestion and add feature flipping.

Comment (sachinrameshn) [reply packages/components/ng-ovh-payment-method/src/components/integration/component/adyen/controller.js]: Its done

File: packages/components/ng-ovh-payment-method/src/components/integration/component/adyen/controller.js [hunk 2]
```diff
@@ -253,4 +254,16 @@ export default class OvhPaymentMethodIntegrationComponentAdyenCtrl {
       callbackParamObject,
     );
   }
+
+  getEnvironment() {
+    return this.ovhFeatureFlipping
+      .checkFeatureAvailability(PAYMENT_ADYEN_LIVE_IN_CONFIG_FEATURE_ID)
+      .then((featureAvailability) =>
+        featureAvailability.isFeatureAvailable(
+          PAYMENT_ADYEN_LIVE_IN_CONFIG_FEATURE_ID,
```
Comment (darsene) [inline packages/components/ng-ovh-payment-method/src/components/integration/component/adyen/controller.js:263]: use routing resolve section to load it

**Review by qpavy — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**